README.md

```diff
@@ -136,3 +136,18 @@ kubectl delete --ignore-not-found thirdpartyresource \
 ```
 
 **The Prometheus Operator collects anonymous usage statistics to help us learning how the software is being used and how we can improve it. To disable collection, run the Operator with the flag `-analytics=false`**
+
+## Development
+
+### Prerequisites
+
+- golang environment
+- minikube (optional)
```

Comment:
`make crossbuild` also requires a docker host as it uses docker under the hood to compile